.projenrc.ts

```diff
@@ -8,6 +8,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   author: "Ben Limmer",
   authorAddress: "hello@benlimmer.com",
   cdkVersion: "2.59.0", // First release of 2023
+  jsiiVersion: "~5.0.0",
```

Comment:
supresses this warning:

```shell
#######################################################################################################
### You are currently using jsii-rosetta@1.x. We recommend upgrading to jsii-rosetta@5.0.x or       ###
### later. This will allow you to use modern TypeScript syntax, and improves compatibility with     ###
### many common dependencies. For example, recent versions of @types/node.                          ###
###                                                                                                 ###
### 5.0.x and subsequent releases of jsii-rosetta use the same major.minor version as the           ###
### TypeScript compiler they are built on. We recommend declaring a minor-pinned (also known as     ###
### "tilde") dependency on jsii-rosetta (e.g: `"jsii-rosetta": "5.0.x"` or                          ###
### `"jsii-rosetta": "~5.0.7"`).                                                                    ###
###                                                                                                 ###
### For more information, see: https://aws.github.io/jsii/compiler-and-rosetta-maintenance/         ###
###                                                                                                 ###
### This warning can be suppressed by setting the JSII_SUPPRESS_UPGRADE_PROMPT environment variable ###
#######################################################################################################
```